src/OrleansRuntime/Messaging/GatewayAcceptor.cs

```diff
@@ -10,18 +11,21 @@ internal class GatewayAcceptor : IncomingMessageAcceptor
         private readonly Gateway gateway;
         private readonly CounterStatistic loadSheddingCounter;
         private readonly CounterStatistic gatewayTrafficCounter;
+        private readonly GlobalConfiguration globalConfig;
 
         internal GatewayAcceptor(
             MessageCenter msgCtr,
             Gateway gateway,
             IPEndPoint gatewayAddress,
             MessageFactory messageFactory,
-            SerializationManager serializationManager)
+            SerializationManager serializationManager,
+            GlobalConfiguration globalConfig)
```

Comment:
I was thinking we should probably move to some kind of "IMultiClusterConfig" eventually - and likewise for other aspects of the configuration